wdl-engine/src/eval/v1/task.rs

```diff
@@ -630,7 +630,16 @@ impl TaskEvaluator {
             "evaluating task"
         );
 
+        // Write input JSON immediately after state initialization
         let mut state = State::new(root, document, task)?;
+
+        // Write the inputs.json file immediately, before any execution
```

Comment:
```suggestion
        // Write the inputs JSON file immediately, before any execution
```

Let's avoid writing the file name in multiple places.